roles/ceph-validate/tasks/check_iscsi.yml

```
@@ -1,4 +1,24 @@
 ---
+- name: set_fact use_new_ceph_iscsi package or old ceph-iscsi-config/cli
+  set_fact:
+    use_new_ceph_iscsi: "{{ (gateway_ip_list | default('0.0.0.0') == '0.0.0.0' and gateway_iqn | default('') | length == 0 and client_connections | default({}) | length == 0 and rbd_devices | default({}) | length == 0) | bool | ternary(true, false) }}"
```

these variables have all a default in `roles/ceph-iscsi-gw/defaults/main.yml`

```suggestion
    use_new_ceph_iscsi: "{{ (gateway_ip_list == '0.0.0.0' and gateway_iqn | length == 0 and client_connections | length == 0 and rbd_devices | length == 0) | bool | ternary(true, false) }}"
```